src/DatesSimple.java

```diff
@@ -0,0 +1,35 @@
+public class DatesSimple {
+    public static void main(String[] args) {
+        int fromy = 2000, fromm = 2, fromd = 3;
+        int toy = 2127, tom = 11, tod = 29;
+
+        System.out.println("**** Original\n" +
+                "From " + fromy + "/" +
+                fromm + "/" + fromd + " to " +
+                toy + "/" + tom + "/" + tod);
+//      pack
+        int period = 0;
+        int resetFromy = fromy - 2000; // adjust the fromy to be in range [0, 127]
+        int resetToy = toy - 2000; // adjust the toy to be in range [0, 127]
+
+        int packedDate = (resetFromy & 0x7f) << 9 | (fromm & 0x0f) << 5 | (fromd & 0x1f);
```

Comment:
Please write a comment explaining what this line of code do - how do you pack those bytes